core/frontend/src/menus.ts

```diff
@@ -98,39 +103,34 @@ const menus = [
         text: 'Used for forwarding UDP NMEA streams into ArduPilot.',
       },
       {
-        title: 'System Information',
-        icon: 'mdi-chart-pie',
-        route: '/tools/system-information',
+        title: 'Ping Sonar Devices',
+        icon: 'mdi-radar',
+        route: '/vehicle/pings',
         advanced: false,
-        text: 'Detailed system status information, CPU, memory, disk, and ethernet status.',
-      },
-      {
-        title: 'Network Test',
-        icon: 'mdi-speedometer',
-        route: '/tools/network-test',
-        show: true,
-        text: 'Test link speed between topside computer and your vehicle.',
+        text: 'Ping devices discovery and management. This allows you to see all detected Ping devices connected to '
+              + 'either your Onboard Computer or its local network.',
       },
       {
-        title: 'Terminal',
-        icon: 'mdi-console',
-        route: '/tools/web-terminal',
+        title: 'Serial-UDP Bridges',
```

Comment:
I just read the description, and I wasn't aware this could do TCP as well. If we expect it to be mostly used with UDP then we can probably leave it, but otherwise I guess we could change it to `Serial-IP Bridges` or something?

Reply:
I believe that only Serial Bridges should work